### PR TITLE
Stop calling a method static if its not static

### LINKED ISSUE #846: newQuery() is not static
`php artisan ide-helper:models --write` adds `query()` and `newQuery()` as static methods in the PHPDoc of the model class.

`query()` is static, but `newQuery()` isn't. However, ide-helper marks them both as static.

This normally wouldn't matter too much, until you use strict PHPStan which states that any usages of `newQuery()` should be static. So `$model->newQuery()` fails PHPStan due to the false PHPDoc added by ide-helper.

I hope that makes sense, but basically `newQuery()` shouldn't be marked as static.

### PR DESCRIPTION
**Pull request by Jamesking56**
Fixes #846

### CODE REVIEW

**Comment by Jamesking56** [comment]
Same test failure as `master`

**Comment by mfn** [comment]
Yeah, I'd say the failed tests are unrelated. Not sure why, the tested console command (ModelsCommand) seems to error out only in Laravel 6 and the tests poorly handle that case. Locally it works for me…

**Comment by mfn** [comment]
Actually no, this PR seems to break it. This is the exception caught in `generateDocs` (but unfortunately poorly handled in the tests):
> Method Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\CustomDate\Models\CustomDate::whereCreatedAt() does not exist
```
#0 /laravel-ide-helper/src/Console/ModelsCommand.php(413): ReflectionMethod->__construct(Object(Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\CustomDate\Models\CustomDate), 'whereCreatedAt')
#1 /laravel-ide-helper/src/Console/ModelsCommand.php(220): Barryvdh\LaravelIdeHelper\Console\ModelsCommand->getPropertiesFromTable(Object(Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\CustomDate\Models\CustomDate))
#2 /laravel-ide-helper/src/Console/ModelsCommand.php(118): Barryvdh\LaravelIdeHelper\Console\ModelsCommand->generateDocs(Array, Array)
#3 [internal function]: Barryvdh\LaravelIdeHelper\Console\ModelsCommand->handle()
#4 /laravel-ide-helper/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(32): call_user_func_array(Array, Array)
#5 /laravel-ide-helper/vendor/laravel/framework/src/Illuminate/Container/Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#6 /laravel-ide-helper/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#7 /laravel-ide-helper/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Array, Object(Closure))
#8 /laravel-ide-helper/vendor/laravel/framework/src/Illuminate/Container/Container.php(590): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Array, Array, NULL)
#9 /laravel-ide-helper/vendor/laravel/framework/src/Illuminate/Console/Command.php(202): Illuminate\Container\Container->call(Array)
#10 /laravel-ide-helper/vendor/symfony/console/Command/Command.php(255): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArrayInput), Object(Mockery_1_Illuminate_Console_OutputStyle))
#11 /laravel-ide-helper/vendor/laravel/framework/src/Illuminate/Console/Command.php(189): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArrayInput), Object(Mockery_1_Illuminate_Console_OutputStyle))
#12 /laravel-ide-helper/vendor/symfony/console/Tester/CommandTester.php(76): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArrayInput), Object(Symfony\Component\Console\Output\StreamOutput))
#13 /laravel-ide-helper/tests/TestCase.php(32): Symfony\Component\Console\Tester\CommandTester->execute(Array)
#14 /laravel-ide-helper/tests/Console/ModelsCommand/CustomDate/Test.php(64): Barryvdh\LaravelIdeHelper\Tests\TestCase->runCommand(Object(Barryvdh\LaravelIdeHelper\Console\ModelsCommand), Array)
#15 /laravel-ide-helper/vendor/phpunit/phpunit/src/Framework/TestCase.php(1408): Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\CustomDate\Test->test()
#16 /laravel-ide-helper/vendor/phpunit/phpunit/src/Framework/TestCase.php(1028): PHPUnit\Framework\TestCase->runTest()
#17 /laravel-ide-helper/vendor/phpunit/phpunit/src/Framework/TestResult.php(691): PHPUnit\Framework\TestCase->runBare()
#18 /laravel-ide-helper/vendor/phpunit/phpunit/src/Framework/TestCase.php(756): PHPUnit\Framework\TestResult->run(Object(Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\CustomDate\Test))
#19 /laravel-ide-helper/vendor/phpunit/phpunit/src/Framework/TestSuite.php(597): PHPUnit\Framework\TestCase->run(Object(PHPUnit\Framework\TestResult))
#20 /laravel-ide-helper/vendor/phpunit/phpunit/src/TextUI/TestRunner.php(621): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#21 /laravel-ide-helper/vendor/phpunit/phpunit/src/TextUI/Command.php(200): PHPUnit\TextUI\TestRunner->doRun(Object(PHPUnit\Framework\TestSuite), Array, true)
#22 /laravel-ide-helper/vendor/phpunit/phpunit/src/TextUI/Command.php(159): PHPUnit\TextUI\Command->run(Array, true)
#23 /laravel-ide-helper/vendor/phpunit/phpunit/phpunit(61): PHPUnit\TextUI\Command::main()
#24 {main}
```

**Comment by mfn** [comment]
(I'll improve this unexpected error handling later)

**Comment by Jamesking56** [comment]
Is this good to merge then or do I need to change something here?

File: .editorconfig [hunk 1]
```diff
@@ -1,15 +0,0 @@
-; This file is for unifying the coding style for different editors and IDEs.
```
Comment (mfn) [inline .editorconfig]: Undo the file removal, please

**Comment by mfn** [comment]
See also https://github.com/barryvdh/laravel-ide-helper/pull/897#issuecomment-595955417

**Pull request closed without merging**